arm/servicebus/models.go

```diff
@@ -453,29 +375,24 @@ func (client TopicListResult) TopicListResultPreparer() (*http.Request, error) {
 		autorest.WithBaseURL(to.String(client.NextLink)))
 }
 
-// TopicProperties is
+// TopicProperties is the Tpoic Properties definition.
 type TopicProperties struct {
-	AccessedAt                          *date.Time               `json:"accessedAt,omitempty"`
-	AutoDeleteOnIdle                    *string                  `json:"autoDeleteOnIdle,omitempty"`
-	EntityAvailabilityStatus            EntityAvailabilityStatus `json:"entityAvailabilityStatus ,omitempty"`
-	CreatedAt                           *date.Time               `json:"createdAt,omitempty"`
-	CountDetails                        *MessageCountDetails     `json:"countDetails,omitempty"`
-	DefaultMessageTimeToLive            *string                  `json:"defaultMessageTimeToLive,omitempty"`
-	DuplicateDetectionHistoryTimeWindow *string                  `json:"duplicateDetectionHistoryTimeWindow ,omitempty"`
-	EnableBatchedOperations             *bool                    `json:"enableBatchedOperations,omitempty"`
-	EnableExpress                       *bool                    `json:"enableExpress,omitempty"`
-	EnablePartitioning                  *bool                    `json:"enablePartitioning,omitempty"`
-	EnableSubscriptionPartitioning      *bool                    `json:"enableSubscriptionPartitioning,omitempty"`
-	FilteringMessagesBeforePublishing   *bool                    `json:"filteringMessagesBeforePublishing,omitempty"`
```

Comment:
Good news on this front, @tombuildsstuff. We've reached out to the service team, and they've agreed to publish these changes as part of a new API Version instead of modify the existing one. (Which is more consistent with the way API Versions are supposed to work.) that means that this change will not be part of V9